zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/processing/IdentitySetupInitializerIT.java

```diff
@@ -82,7 +82,7 @@ void shouldInitializeIdentity(final boolean enableAuthorizations) {
     assertTrue(passwordMatches);
 
     final var createdRole = RecordingExporter.roleRecords(RoleIntent.CREATED).getFirst().getValue();
-    Assertions.assertThat(createdRole).hasName(IdentitySetupInitializer.DEFAULT_ROLE_NAME);
+    Assertions.assertThat(createdRole).hasName("Admin");
```

Comment:
🔧 We should also extend this to check for the `rpa` and `connectors` role

Reply:
Done in 84163d81268f689cc23641e690efa7abf1068c19